test/functional/feature_blocksxor.py

```diff
@@ -54,7 +53,7 @@ def run_test(self):
             match=ErrorMatch.PARTIAL_REGEX)
 
         self.log.info("Delete XOR key, restart node with '-blocksxor=0', check blk*.dat/rev*.dat file integrity")
-        os.remove(node.blocks_path / 'xor.dat')
+        os.remove(node.blocks_key_path)
```

Comment:
nit: could put this already into the commit that introduces the blocks_key_path property to TestNode.

Reply:
Yup, this is more coherent.  Updated.